ansible/roles/lvm/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
 
 - name: Create partition on block storage volume
   sudo: yes
-  shell: echo -e "o\nn\np\n1\n\n\nt\n1\n8e\nw" | fdisk {{ item.item }}
```

Comment:
@dannyroberts this worked fine last time I deployed (ctrls) did you use 14.04 to test? fdisk changed this behavior (smarter autoselect partition number) in a commit, this is why our LVM is a bit risky or dangerous, for example it would fail if server language is set to French.

Reply:
Really? I think @calellowitz and I checked enikshay (ctrls) and saw some FAT12-formatted drives there as well, presumably for the same reason. When you said it worked fine, do you mean it didn't halt mid-way and you did end up with a consolidated drive, or do you mean you're sure they were formatted as Linux (or whatever the target format is)?

Reply:
i see... yeah, that's what i meant by "it worked", you're right, partitions were created with FAT12 type, but just so we're clear they were all correctly formatted as ext4. as far as I understand, otherwise they would have failed.

Reply:
Ah, interesting. I wasn't making a distinction between those two things. Sounds like this change is still technically an improvement, though, even if it's inconsequential in the scheme of things, right?

Reply:
yeah, absolutely